src/alire/alire-utils-user_input-query_config.adb

```diff
@@ -36,7 +36,7 @@ package body Alire.Utils.User_Input.Query_Config is
 
    function User_Name return String
    is (Config_Or_Query_String (Config_Key => "user.name",
-                               Question   => "User name",
+                               Question   => "Please enter your user name:",
```

Comment:
It should probably `Please enter your name:`, there no "user name" for Alire.

Reply:
Right, oof.